index.bs

```diff
@@ -157,24 +157,26 @@ To <dfn abstract-op local-lt="parse header">obtain a response's embedder policy<
 
 1.  Let |policy| be "`unsafe-none`".
 
-2.  Let |header| be the result of [=header list/getting=] `Cross-Origin-Embedder-Policy` from
+2.  Let |value| be the result of [=header list/get a structured header=]
+    `Cross-Origin-Embedder-Policy` and "`item`" from
     |response|'s [=response/header list=].
 
-3.  If |header| is not `null`:
+3.  If |value| is not `null` and is not failure:
 
-    1.  Set |header| to the result of [=isomorphic decoding=] |header|.
+    1. Let |parsed policy| be |value|[0].
 
-    2.  Let |parsed policy| be the result of executing the [$Structured Header parsing algorithm$]
-        with <var ignore>input_string</var> set to |header|, and <var ignore>header_type</var> set
-        to "`item`".
+    2. Let |parameters| be |value|[1].
 
-        If parsing fails, set |parsed policy| to "`unsafe-none`".
+    3. If |parsed policy| is not a [=structured header/token=], then return |policy|.
 
-        ISSUE(httpwg/http-extensions#662): The ASCII requirements of Structured Headers are
-        somewhat unclear to me. Do we need to check whether |header| is an [=ASCII string=] before
-        passing it into the parsing algorithm?
+    4. If |parameters| is not empty, then return |policy|.
 
-    3.  If |parsed policy| is "`require-corp`", set |policy| to "`require-corp`".
+        ISSUE: This step is expected to change when the Structured Headers for HTTP specification is
+        more stable. See [whatwg/html issue #5172](https://github.com/whatwg/html/issues/5172).
+        [[I-D.ietf-httpbis-header-structure]]
```

Comment:
I'm fine with landing this just to correct the current text, but I think we're at the point where the IETF WG considers SH pretty close to done. I'd be surprised if something like parameterized items changed substantially at this point.

Reply:
(So, LGTM from my side. If @annevk's also happy, land it. :))